alert-policies/redis-otel/HighMemoryUsage.yml

```
@@ -0,0 +1,31 @@
+name: Memory Utilization is too high
```

```suggestion
name: Memory utilization is too high
```
As per NR standards we need to go with sentence case format.